src/pytest_html/basereport.py

```diff
@@ -284,7 +284,23 @@ def _process_report(self, report, duration, processed_extras):
         ]
         cells = [
             f'<td class="col-result">{outcome}</td>',
-            f'<td class="col-testId">{test_id}</td>',
+            f"""<td class="col-testId">{test_id}
+                <button class="copy-btn"
+                        data-test-id="{test_id}"
+                        title="Copy test ID"
+                        aria-label="Copy test ID">
+                    <svg xmlns="http://www.w3.org/2000/svg"
```

Comment:
how feasible is it to use U+1F4CB instead of the svg - i'd much prefer the character - but i have no idea how well supported that is